controllers/clusterresources/awsencryptionkey_controller.go

```diff
@@ -129,6 +129,7 @@ func (r *AWSEncryptionKeyReconciler) handleCreate(
 		)
 
 		encryptionKey.Status = *encryptionKeyStatus
+		encryptionKey.Status.State = models.CreatedStatus
```

Comment:
try the thing I suggested by replacing `statusCode` with `state` or vice versa, as you prefer.

Reply:
Started using existing `Status` field